src/spring-cloud/azext_spring_cloud/_help.py

```diff
@@ -542,3 +542,46 @@
         - name: Unbind an app to Application Configuration Service.
           text: az spring-cloud application-configuration-service unbind --app MyApp -s MyService -g MyResourceGroup
 """
+
+helps['spring-cloud build-service buildpacks-binding'] = """
+    type: group
+    short-summary: (Enterprise Tier Only) Commands to manage Buildpacks Binding
+"""
+
+helps['spring-cloud build-service buildpacks-binding create'] = """
+    type: command
+    short-summary: Create a buildpacks binding.
+    examples:
+        - name: Create a buildpacks binding without properties or secrets.

```

Comment:
I suppose all help need to be updated to newest version.

Reply:
Updated